Services/Seed.cs

```diff
@@ -147,51 +147,30 @@ public class Seed : ISeed
 
             foreach (var group in template.Groups)
             {
-                try
-                {
-                    await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
-                }
-                catch (Exception ex)
-                {
-                    this.log.Error($"Failed to seed default group {group.DisplayName}", () => new { group, ex.Message });
-                }
+                await this.storage.UpdateDeviceGroupAsync(group.Id, group, "*");
             }
 
             foreach (var rule in template.Rules)
             {
-                try
-                {
-                    await this.telemetryClient.UpdateRuleAsync(rule, "*");
-                }
-                catch (Exception ex)
-                {
-                    this.log.Error($"Failed to seed default rule {rule.Description}", () => new { rule, ex.Message });
-                }
+                await this.telemetryClient.UpdateRuleAsync(rule, "*");
             }
 
-            try
-            {
-                var simulationModel = await this.simulationClient.GetSimulationAsync();
+            var simulationModel = await this.simulationClient.GetSimulationAsync();
 
-                if (simulationModel != null)
-                {
-                    this.log.Info("Skip seed simulation since there is already one simuation", () => new { simulationModel });
-                }
-                else
-                {
-                    simulationModel = new SimulationApiModel
-                    {
-                        Id = "1",
-                        Etag = "*"
-                    };
-
-                    simulationModel.DeviceModels = template.DeviceModels.ToList();
-                    await this.simulationClient.UpdateSimulation(simulationModel);
-                }
+            if (simulationModel != null)
+            {
+                this.log.Info("Skip seed simulation since there is already one simulation", () => new { simulationModel });
             }
-            catch (Exception ex)
+            else
             {
-                this.log.Error("Failed to seed default simulation", () => new { ex.Message });
+                simulationModel = new SimulationApiModel
+                {
+                    Id = "1",
+                    Etag = "*",
+                    DeviceModels = template.DeviceModels.ToList()
+                };
+
+                await this.simulationClient.UpdateSimulation(simulationModel);
             }
         }
```

Comment:
Please add a log statement for "Seed data generation completed successfully."

Reply:
A similar log message was already there in Seed.TrySeedAsync, line 73